### PR TITLE
LPS-157652

### PR DESCRIPTION
**Pull request by carolmariaabb**
https://issues.liferay.com/browse/LPS-157652

### CODE REVIEW

**Comment by carolmariaabb** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java [hunk 1]
```diff
@@ -109,8 +110,7 @@ public ObjectView addObjectView(
 				user, objectView, objectViewFilterColumns));
 		objectView.setObjectViewSortColumns(
 			_addObjectViewSortColumns(
-				user, objectView.getObjectViewId(), objectViewColumns,
-				objectViewSortColumns));
+				user, objectView, objectViewColumns, objectViewSortColumns));
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java:113]: Same order as `_addObjectViewFilterColumns`

**Review by rodrigopaulino — APPROVED** [review]
LGTM! Thanks for fixing this so fast, @carolmariaabb.
Amazing job! 👏🏻

**Comment by rodrigopaulino** [comment]
ci:forward

**Comment by brianchandotcom** [comment]
@carolmariaabb merged to master and the temp branch.

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:close

**Pull request closed without merging**